session.rst

```diff
@@ -1328,6 +1328,47 @@ has to return an integer which will be used as TTL.
             // Inject whatever dependencies you need to be able to resolve a TTL for the current session
             ->args([service('security')]);
 
+
+.. _session-passing-session-parameter:
+
+Passing a session parameter to a controller argument
+----------------------------------------------------
+
+A possibility is to pass a session parameter to a controller argument.
+Let's say you declare the following DTO ::
+
+    namespace App\Model;
+
+    class PreferenceDTO
```

Comment:
Not sure we need the DTO suffix here. Maybe just `Preferences`? or `UserPreferences`?

Reply:
Renamed to UserPreferences to show more that it's related to the current user and not the system